src/aggregation/agg_result.rs

```diff
@@ -137,6 +137,17 @@ impl BucketResult {
     }
 }
 
+/// This is the wrapper of buckets entries, which can be vector or hashmap
+/// depending on if it's keyed or not.
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+#[serde(untagged)]
+pub enum BucketEntries<T> {
+    /// Vector format bucket entries
+    Vec(Vec<T>),
+    /// HashMap format bucket entries
+    HashMap(HashMap<String, T>),
```

Comment:
FnvHashMap would be better here imo

Reply:
I see!  updated! https://github.com/quickwit-oss/tantivy/pull/1424/commits/80a1418284b140a833fe35aba688d0b14197a9b2